src/forms/TextInput.jsx

```diff
@@ -5,79 +5,62 @@ import cx from 'classnames';
 /**
  * @module TextInput
  */
-class TextInput extends React.Component {
+function TextInput(props) {
```

Comment:
Hmmm, yeah we've been doing it the es6 way as we've been migrating to stateless. I see the style guide says boo to it. I vote for the es6, but I'll defer to the collective on which way we go. 😃